tests/TestRender.php

```diff
@@ -4,39 +4,36 @@
 class TestRender {
     protected ?Format $format = null;
 
+    protected Config $config;
+
     public function __construct(
         ?Format $format,
-        array $opts,
-        ?array $extra = [],
-        ?array $indices = []
+        Config $config,
```

Comment:
I have no strong opinion on this but it might be better that way. I'll make a note of this and will change this the next time I refactor testing related files.